heltour/tournament/models.py

```diff
@@ -1816,6 +1823,9 @@ def save(self, *args, **kwargs):
             if alt.priority_date_override is None or alt.priority_date_override < current_date:
                 alt.priority_date_override = current_date
                 alt.save()
+                
+        if self.games_missed >= 2 and self.initial_games_missed < 2:
+            self._set_unavailable_for_season()
```

Comment:
I'm not certain how the `initial_games_missed` interacts with the `games_missed`? Does `games_missed` always include all of the `initial_games_missed`?  If so, does this mean that missing 2 games (1 at the beginning and 1 elsewhere) you'll get set to unavailable for the entire season?

Reply:
`games_missed` is a property of a season player. It is used to determine if a season player has a yellow or red card. If a season player has `games_missed = 1` it means that they have a yellow card in the respective season. If they have `season_games >= 2` they have received a red card. The idea is that once a player receives a red card, they are set unavailable for the remainder of the season. So far the team league mods had to do that manually: Each time a player received a red card (missing games went from 1 to 2) we had to notice that and had to set the player unavailable for the remaining rounds. 

Now, the only way that `games_missed` could be lower than `initial_games_missed` is when a moderator or a succesful no-show appeal reduce the number of games missed. In that case we would still have to ask the player to update their availability manually, as we are not able to determine which rounds we should set back to avialable (i.e. which rounds heltour set to unavailable because of the red card and which rounds the player had set to unavailable already before).

Not sure if that answers your question. TLDR: `games_missed` always includes `initial_games_missed`, unless the player gets a card revoked by moderator action. Either directly on the season player object or by accepting a no-show appeal.

Reply:
The `season_games >= 2` should be `games_missed >= 2` right?

Reply:
Oh, I missed that `initial_games_missed` is a cached version of the value from when the object is created. 

We could probably make this slightly clearer by refactoring that into a method of some sort:
`def just_received_red_card(self):` and have it check them, with a minor comment to that effect. 

But it's not important, I'm merging.